### DISCUSSION TITLE
Is there anywhere users share their custom themes/templates/color styles? Or a repository of some sort?

### DISCUSSION THREAD

**Discussion by Foxxsayy** [freeplane/freeplane#1051] [Help]
I've been trying to figure out Dark Mode, and I think it's just done through styles. But I'd like to see what others have come up with, and I wanted to know if there was any sort of repository or sharing center for themes?

**Comment by macmarrum** [comment]
Please have a look at https://github.com/freeplane/freeplane/discussions/categories/shared-mind-maps

**Comment by sifran-github** [comment]
I can post my dark theme template. but... whats the difference between:
https://github.com/freeplane/freeplane/discussions/categories/shared-mind-maps
https://github.com/freeplane/freeplane/discussions/categories/show-and-tell

where should i post it?

**Comment by sifran-github** [comment]
posted here: https://github.com/freeplane/freeplane/discussions/1053
